### PR TITLE
Installation instructions

### LINKED ISSUE #67: Add installation instructions
How do I install this so that I can use the widgets in this package in Orange?

### PR DESCRIPTION
**Pull request by ajdapretnar**
Fixes #67.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,44 @@
...
+    # Setup the add-on
+    python setup.py install
+
+To install Orange in editable/development mode, run
+
+    pip install -e .
+
+Alternatively, you can install the add-on with pip:
+
+	pip install orange3-prototypes
```
Comment (kernc) [inline README.md]: Replace tab with spaces.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,44 @@
...
+
+
+## OpenCV dependency
```
Comment (kernc) [inline README.md]: I would nest this under Installing.

Comment (kernc) [inline README.md]: ftfy.

**Comment by nikicc** [comment]
I'm not sure that `opencv-python` is the way to go here. It causes segmentation faults (at least on OS X) when closing the widget. Working with opencv package from anaconda works OK.

**Comment by kernc** [comment]
The issue might still be on our end. Let's not forget who coded the widget ...

**Comment by nikicc** [comment]
Sure, but untill we make it work with `opencv-python`, do we really want to instruct users to install it?

**Comment by bernardofn** [comment]
After running the installation instructions, I can now have the webcam widget but it gives me an error when taking the picture with the webcam.


Exception: | cv2.error: /Users/travis/build/skvark/opencv-python/opencv/modules/objdetect/src/cascadedetect.cpp:1698: error: (-215) !empty() in function detectMultiScale
-- | --
Module: | orangecontrib.prototypes.widgets.owwebcamcapture:115
Widget Name: | Webcam Capture
Widget Module: | orangecontrib.prototypes.widgets.owwebcamcapture:162
Widget Scheme: | /var/folders/bm/d2x2037j5hqdcg86p8hqf1nw0000gn/T/ows-pz7f8gsi.ows.xml
Version: | 3.7.1
Environment: | Python 3.6.1 on Darwin 16.7.0 Darwin Kernel Version 16.7.0: Thu Jun 15 17:36:27 PDT 2017; root:xnu-3789.70.16~2/RELEASE_X86_64 x86_64
Installed Packages: | AnyQt==0.0.8, Bottleneck==1.2.0, CacheControl==0.12.3, CommonMark==0.7.3, Orange3-Educational==0.1.4, Orange3-ImageAnalytics==0.1.10, Orange3-Prototypes==0.7.0, Orange3-Recommendation==0.1.3, Orange3==3.7.1, Pillow==4.3.0, PyQt5==5.9, beautifulsoup4==4.6.0, certifi==2017.11.5, chardet==3.0.4, dill==0.2.6, docutils==0.13.1, future==0.16.0, h2==2.6.2, hpack==3.0.0, hyper==0.7.0, hyperframe==3.2.0, idna==2.6, joblib==0.11, keyring==10.3.1, keyrings.alt==2.2, lockfile==0.12.2, mock==2.0.0, msgpack-python==0.4.8, numpy==1.12.1, olefile==0.44, opencv-python==3.3.0.10, pbr==3.1.1, pip==9.0.1, pyqtgraph==0.10.0, requests==2.18.4, scikit-learn==0.18.2, scipy==0.19.1, serverfiles==0.2.1, setuptools-git==1.2, setuptools==36.8.0, sip==4.19.3, six==1.10.0, urllib3==1.22, wheel==0.30.0, xlrd==1.0.0
Machine ID: | 154505279153765
Stack Trace: | Traceback (most recent call last):  File "/Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/orangecontrib/prototypes/widgets/owwebcamcapture.py", line 162, in capture_image    (self.clip_aspect_frame(frame), '_aspect', self.Output.SNAPSHOT_ASPECT)):  File "/Applications/Orange3.app/Contents/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/orangecontrib/prototypes/widgets/owwebcamcapture.py", line 115, in clip_aspect_frame    cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY))cv2.error: /Users/travis/build/skvark/opencv-python/opencv/modules/objdetect/src/cascadedetect.cpp:1698: error: (-215) !empty() in function detectMultiScale
Local Variables: | OrderedDict([('ASPECT_RATIO', 0.8),             ('frame',              array([[[146, 134, 119],        [149, 136, 121],        [141, 133, 120],        ...,         [ 76,  88,  96],        [ 73,  88,  91],        [ 73,  88,  91]],       [[148, 135, 120],        [148, 135, 120],        [142, 134, 121],        ...,         [ 76,  88,  96],        [ 73,  88,  91],        [ 71,  87,  90]],       [[144, 134, 119],        [144, 134, 119],        [143, 135, 120],        ...,         [ 74,  87,  97],        [ 63,  85,  87],        [ 65,  87,  90]],       ...,        [[ 26,  20,  18],        [ 26,  20,  18],        [ 27,  16,  15],        ...,         [ 78,  67,  75],        [ 82,  65,  71],        [ 83,  66,  72]],       [[ 26,  19,  20],        [ 26,  19,  20],        [ 27,  15,  16],        ...,         [ 86,  64,  62],        [ 79,  58,  51],        [ 90,  69,  62]],       [[ 25,  18,  19],        [ 27,  20,  21],        [ 33,  21,  22],        ...,         [ 92,  70,  67],        [ 86,  65,  58],        [ 92,  71,  64]]], dtype=uint8))])

**Pull request merged**